base/compiler/optimize.jl

```diff
@@ -121,6 +121,93 @@ function ir_to_codeinf!(opt::OptimizationState)
     return src
 end
 
+##################
+# EscapeAnalysis #
+##################
```

Comment:
We shouldn't define EA in `Core.Compiler`, otherwise some symbols for lattice ops will collide with those for inference and will lead to significant performance regression.

Reply:
Basically resolved by https://github.com/aviatesk/EscapeAnalysis.jl/pull/59 and corresponding commit bd0bdb7.